tests/scripts/deploy_admission_controller.sh

```diff
@@ -11,7 +11,24 @@ function cleanup() {
   kubectl delete -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
   set -e
 }
-trap cleanup SIGINT ERR
+
+function error_log() {
+  set +e -x
+  kubectl get validatingwebhookconfigurations cert-manager-webhook -o yaml | grep caBundle
+  kubectl get mutatingwebhookconfigurations cert-manager-webhook -o yaml | grep caBundle
+  kubectl -n rook-ceph get certificate
+  kubectl -n rook-ceph get secret | grep rook-ceph-admission-controller
+  kubectl -n rook-ceph get validatingwebhookconfigurations.admissionregistration.k8s.io 
+  kubectl describe validatingwebhookconfigurations.admissionregistration.k8s.io cert-manager-webhook 
+  kubectl describe validatingwebhookconfigurations.admissionregistration.k8s.io rook-ceph-webhook 
+  kubectl -n cert-manager logs deploy/cert-manager-webhook --tail=10
```

Comment:
No need to output the webhook logs. May make the test logs verbose and we always get the specific errors messages which caused the failure . Can we instead check if all the major pieces for running the admission webhooks are in place and in healthy state.

Reply:
what other major pieces?

Reply:
pods, services that rook creates, status of rook operator (provided it is up in tests when we execute this) ? think you may need to add the issuer here too.

Reply:
Also we can remove caBundle output for the webhhoks as it will mean nothing when we look at it in the logs. If the caBundles are missing, then the validations which rook will perform will fail.
I think just checking the status may be enough otherwise you may want to add logs when you print the output

Reply:
> pods, services that rook creates, status of rook operator (provided it is up in tests when we execute this) ? think you may need to add the issuer here too.

yes for Issuer, for the pods and services that rook creates, it will be created after this script is done.

Reply:
> Also we can remove caBundle output for the webhhoks as it will mean nothing when we look at it in the logs. If the caBundles are missing, then the validations which rook will perform will fail.
> I think just checking the status may be enough otherwise you may want to add logs when you print the output

sounds good, will remove